ui/src/app/store/pod/slice.ts

```diff
@@ -23,7 +23,21 @@ type GetProjectsMeta = {
   type: PodType;
 };
 
-type UpdatePod = Partial<Omit<Pod, "tags"> & { tags: string }>;
+type CreateParams = {
+  name?: Pod["name"];
+  tags?: string;
+  zone?: Pod["zone"];
+  pool?: Pod["pool"];
+  cpu_over_commit_ratio?: Pod["cpu_over_commit_ratio"];
+  memory_over_commit_ratio?: Pod["memory_over_commit_ratio"];
+  default_storage_pool?: Pod["default_storage_pool"];
+  default_macvlan_mode?: Pod["default_macvlan_mode"];
+};
+
+type UpdateParams = CreateParams & {
+  [PodMeta.PK]: Pod[PodMeta.PK];
+  tags?: string;
```

Comment:
Looks like this is already included in `CreateParams`?

Reply:
Oh yeah, thanks.